File: Week_3/problem_3-Conversion.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219856/problem/G

#include <iostream>
#include <cstring>

using namespace std;

int main() {
    char s[100001];
    cin >> s;
    // cout << s << endl;

    for (int i = 0; i < strlen(s); i++) {
        if (s[i] == ',') {
            s[i] = ' ';
        }

        if (s[i] >= 'A' && s[i] <= 'Z') {
            // Convert uppercase to lowercase
            s[i] = s[i] + 32;  
        } else if (s[i] >= 'a' && s[i] <= 'z') {
            // Convert lowercase to uppercase
            s[i] = s[i] - 32;  
        }
    }

    cout << s;

    return 0;
}

File: Week_3/problem_3-Conversion.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219856/problem/G

s = input()

# Replace commas with spaces and switch the case of letters
result = []
for char in s:
    if char == ',':
        result.append(' ')
    elif 'A' <= char <= 'Z':
        # Convert uppercase to lowercase
        result.append(char.lower())  
    elif 'a' <= char <= 'z':
        # Convert lowercase to uppercase
        result.append(char.upper())  
    else:
        result.append(char)

# Join the list of characters back into a string and print the result
print("".join(result))
